src/neuroglancer/datasource/graphene/frontend.ts

```diff
@@ -702,3 +1002,317 @@ class SliceViewPanelChunkedGraphLayer extends SliceViewPanelRenderLayer {
+    activation.registerDisposer(() => {
+      resetMulticutDisplay();
+      displayState.baseSegmentHighlighting.value = priorBaseSegmentHighlighting;
+      displayState.segmentStatedColors.value.assignFrom(priorSegmentStatedColors);
+      segmentationGroupState.focusSegments.assignFrom(priorFocusSegments);
+      displayState.highlightColor.value = priorHighlightColor;
+    });
+
+    const resetMulticutDisplay = () => {
+      resetTemporaryVisibleSegmentsState(segmentationGroupState);
+      displayState.showFocusSegments.value = false;
+      displayState.segmentStatedColors.value.clear(); // TODO, should only clear those that are in temp sets
+      segmentationGroupState.focusSegments.clear();
+      displayState.highlightColor.value = undefined;
+    };
+
+    const updateMulticutDisplay = () => {
+      resetMulticutDisplay();
+      activeGroupIndicator.classList.toggle('blueGroup', multicutState.blueGroup.value);
+
+      const focusSegment = multicutState.focusSegment.value;
+      if (focusSegment === undefined) return;
+
+      displayState.baseSegmentHighlighting.value = true;
+      displayState.highlightColor.value = multicutState.blueGroup.value ? BLUE_COLOR : RED_COLOR;
+      segmentsState.useTemporaryVisibleSegments.value = true;
+      segmentsState.useTemporarySegmentEquivalences.value = true;
+      displayState.showFocusSegments.value = true;
+
+      // add to focus segments and temporary sets
+      segmentationGroupState.focusSegments.add(focusSegment);
+      segmentsState.temporaryVisibleSegments.add(focusSegment);
+
+      for (const segment of multicutState.segments) {
+        segmentsState.temporaryVisibleSegments.add(segment);
+        segmentationGroupState.focusSegments.add(segment);
+      }
+
+      // all other segments are added to the focus segment equivalences
+      for (const equivalence of segmentsState.segmentEquivalences.setElements(focusSegment)) {
+        if (!segmentsState.temporaryVisibleSegments.has(equivalence)) {
+          segmentsState.temporarySegmentEquivalences.link(focusSegment, equivalence);
+        }
+      }
+
+      // set colors
+      for (const segment of multicutState.redSegments) {
+        displayState.segmentStatedColors.value.set(segment, RED_COLOR_PACKED);
+      }
+      for (const segment of multicutState.blueSegments) {
+        displayState.segmentStatedColors.value.set(segment, BLUE_COLOR_PACKED);
+      }
+    };
+
+    updateMulticutDisplay();
+
+    activation.registerDisposer(multicutState.changed.add(updateMulticutDisplay));
+
+    activation.bindAction('swap-group', event => {
+      event.stopPropagation();
+      multicutState.swapGroup();
+    });
+
+    activation.bindAction('set-anchor', event => {
+      event.stopPropagation();
+      
+      const {segmentSelectionState: {baseValue, value}} = this.layer.displayState;
+      if (!baseValue || !value) return; // could this happen or is this just for type checking
```

Comment:
If the user triggers this action while not hovering over a segment, then this condition will occur.